docs/source/user/urls.rst

```diff
@@ -207,7 +207,7 @@ default workspace.
 Workspace File Format
 ---------------------
 
-A workspace file in a JSON file with a specific spec.
+A workspace file is a JSON file with a specific spec.
```

Comment:
Of note this is fixed on 4.2.x and rewritten to: "A workspace file is a JSON file that contains one object with two required top-level keys, data, and metadata."